pkg/platform/runtime/buildscript/json.go

```diff
@@ -0,0 +1,72 @@
+package buildscript
```

Comment:
If you have a concrete suggestion for me, I'm open to hearing it. Otherwise it's just vague hand-waving to me. I've chosen to break up the buildscript package into logical files dealing with buildscripts, buildexpressions, and json. I didn't want to have one monolithic buildscript.go file.

Reply:
The basic suggestion would be to have separate files for Assignment, Value, File, etc. Again, organizing by type rather than purpose (where JSON is a purpose). I'm not directly suggesting that. Rather, just grouping methods with the type would be sufficient in my mind. If this line of thinking still doesn't feel right to you, I'm happy to accept as-is.

Reply:
I see. I am not in favor of that since build scripts are still in flux and would probably result in a lot of file churn. It's nice to have all of these structs present in one file for making quick changes.